packages/dataframe/dataframe.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+name:         "dataframe"
```

Comment:
Name and version are redundant, opam will override them with the name and version from the path

```suggestion
```